packages/@react-aria/overlays/test/useModal.test.js

```diff
@@ -88,4 +88,102 @@ describe('useModal', function () {
     res.rerender(<Example />);
     expect(rootProvider).not.toHaveAttribute('aria-hidden');
   });
+
+  it('can specify a different container from the default document.body', function () {
```

Comment:
Is it possible to check that the "alternateContainer" is the parent of the root provider here?

Reply:
good catch, yeah, we should check that